### DISCUSSION TITLE
Cannot connect Azure to Github

### DISCUSSION THREAD

**Discussion by Anneqxy** [microsoft/SATechnicalOnboarding#92] [Questions]
<img width="1180" alt="截屏2024-01-17 下午6 22 45" src="https://github.com/microsoft/SATechnicalOnboarding/assets/113318686/cff3a42b-f403-41f3-9c55-d2a563508eb6">
I am in the step 4 of Azure Static Web Apps. There is no "Azure for Students" on my computer. How to deal with that situation? Does it mean I missed some steps?
![Uploading 截屏2024-01-17 下午6.25.33.png…]()

**Comment by Anneqxy** [comment]
![Uploading 截屏2024-01-17 下午6.25.33.png…]()

**Reply by AnthonyByansi** [reply]
Hi @Anneqxy It indicates you don't have any subscription. 

You need to make sure you have the credits on your portal. Head to [Azure for students](https://azure.microsoft.com/free/students/?WT.mc_id=%3Fwt.mc_id%3Dstudentamb_260352) and redeem your $100 credit

Once that is done, `DEFAULT DIRECTORY` will be displayed as your subscription. 

If you already attained the Alpha process, then you don't have to go through that long process, instead, redeem you Azure Monthly credits from the [Visual Studio Subscription Home Page](https://my.visualstudio.com/) using your MLSA account
![Azure monthly](https://github.com/microsoft/SATechnicalOnboarding/assets/101401469/f03523a6-17e6-4464-abdc-2d3f4a2b7444)
